docs/src/content/docs/user-roles/assign-roles.mdx

```diff
@@ -0,0 +1,245 @@
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN
+{
+  "results": [
+    "user2@test.ca",
+    "user1@test.ca"
+  ]
+}
+```
+
+
+## Approve or reject a user that has requested access <Badge text="Site admin" variant="caution" />
+
+This can only be performed by a site admin. Unauthorized users can request access by clicking a button in the CanDIG Data portal. They will only see this button if they are unauthorized and are not currently on the pending users list. Clicking this button causes the user to be added to the `pending users` list. A site admin then needs to approve these users so they can become [CanDIG Authorized Users](roles-overview#1-candig-authorized-user). [This diagram](roles-overview#request-flow) demonstrates this process visually.
+
+First [get a token](#getting-a-token), then:
+
+### List pending users
+
+1. Check to see what users are on the pending users list:
+
+```bash
+curl --request GET \
+  --url $CANDIG_URL'/ingest/user/pending' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN
+```
+
+e.g. response:
+```bash
+{
+  "results": [
+    "user1@test.ca",
+    "user2@test.ca"
+  ]
+}
+```
+
+### Approve pending users
+
+2. POST to the `/user/pending` endpoint, either with a single `user_id`:
+
+```bash
+curl --request POST \
+  --url $CANDIG_URL'/ingest/user/pending/user1@test.ca' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN
+```
+
+If successful, you should get a response such as:
+```bash
+{
+  "message": "User user1@test.ca has been approved"
+}
+```
+
+Or a list of ids
+
+```bash
+curl --request POST \
+  --url $CANDIG_URL'/ingest/user/pending' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN \
+  -d '["user2@test.ca", "user1@test.ca"]'
+```
+
+Where a successful response should be something like:
+
+```bash
+{
+  "approved": [
+    "user2@test.ca",
+    "user1@test.ca"
+  ]
+}
+```
+
+### Reject pending users 
+
+To reject users that have requested access, use the `DELETE` method, on the same endpoint. 
+
+:::caution
+`DELETE` clears the whole list of pending users so be sure to approve valid users before using the `DELETE` method to clear any remaining users.
+:::
+
+e.g.
+
+```bash
+curl --request DELETE \
+  --url $CANDIG_URL'/ingest/user/pending' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer '$TOKEN \
+  -d '["user2@test.ca", "user1@test.ca"]'
+```
+
+The response should show that the pending users list is now empty, i.e.:
+
+```bash
+{
+  "pending_users": {}
+}
+```
+
+## Revoke CanDIG Authorized user status <Badge text="Site admin" variant="caution" />
+
+This can only be done by a Site admin. If a user has CanDIG Authorized user status that needs to be revoked, use the `DELETE` method on the user endpoint.
+
```

Comment:
Note that if a user loses access to the credentials they use to log into CanDIG (for example, if they leave their job at the affiliated institution), they will no longer be able to access the site.